Primo.js

```diff
@@ -318,6 +382,9 @@ function importPNX(text) {
 	item.edition = ZU.xpathText(doc, '//display/edition');
 	
 	var subjects = ZU.xpath(doc, '//search/subject');
+	if(!subjects.length) {
+		subjects = ZU.xpath(doc, '//display/subject');
+	}
```

Comment:
I would like to suggest to switch the logic here, i.e.first try to receive the subjects from the display part and just afterwards use the search part. I think it is common to add more writings of the same subject for search improvements, as we do for example in our primo catalogue, see e.g. http://primo.bib.uni-mannheim.de/MAN_UB:MAN_ALEPH001466796 . @adam3smith @aurimasv : What do you think? One part is from a previous commit.